### PR TITLE
Remove plugin wip

### PR DESCRIPTION
**Pull request by elchocarrero**
## Hey, I just made a Pull Request!

Adds remove-plugin cli command
#### :heavy_check_mark: Checklist

- [ ] All tests are passing `yarn test`
- Some errors in plugins trying to import errorApiRef
- [ ] Screenshots attached (for UI changes)
- [ ] Relevant documentation updated
- Need to update documentation
- [ ] Prettier run on changed files
- [X] Tests added for new functionality
- Tests are still WIP
- [X] [CODEOWNERS](./CODEOWNERS) updated (for new stuff)
- When removing plugin, plugin codeowners statement is removed from .github/CODEOWNERS
- [ ] Regression tests added for bug fixes

### CODE REVIEW

**Comment by Rugvip** [comment]
Hi @braulio-balanza, is this ready for review? Would love to get this in

**Comment by braulio-balanza** [comment]
Hi! The problem I am having is that since the command basically undoes everything create-plugin does, I keep having to modify it. Any pointers on how I could proceed?

**Comment by Rugvip** [comment]
@braulio-balanza best to do is to get it in sync with current master and then get it merged. Whoever updates the plugin create process should then update this too. We'll also make use of this in the e2e test to make sure it stays in sync.

**Comment by braulio-balanza** [comment]
Ok, let me reflect the latest changes and I will notify you!

**Comment by braulio-balanza** [comment]
I feel that this pull request is ready for review. I added the unit tests of the most important features. I am using some methods from createPlugin.ts in the tests, which is not preferable but made writing the tests much easier.

File: packages/cli/src/commands/remove-plugin/removePlugin.test.ts [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+}
+
+describe('removePlugin', () => {
+  describe('Remove Plugin Dependencies', () => {
+    const appPath = paths.resolveTargetRoot('packages', 'app');
+    const githubDir = paths.resolveTargetRoot('.github');
+    it('removes plugin references from /packages/app/package.json', async () => {
+      // Set up test
+      const packageFilePath = path.join(appPath, 'package.json');
+      const testFilePath = path.join(appPath, 'test.json');
```
Comment (Rugvip) [inline packages/cli/src/commands/remove-plugin/removePlugin.test.ts:84]: Not crazy about the tests reading/writing to the repo itself. Would prefer things being tested in a temporary directory. And also move to fs mock eventually :p

**Review by Rugvip — DISMISSED** [review]
Thanks for the updates to the latest cli helpers! :grin:

Looks good! 😄

**Comment by braulio-balanza** [comment]
Yeah, I was thinking the same at the end, but I just wanted to commit. Let me do the changes, it shouldn't be that difficult

**Review by Rugvip — APPROVED** [review]
Thanks for the update, ready to merge! 👍

**Pull request merged**